composer.json

```diff
@@ -9,14 +9,16 @@
         {"name": "Benjamin Eberlei", "email": "kontakt@beberlei.de"},
         {"name": "Steve Müller", "email": "st.mueller@dzh-online.de"}
     ],
-    "require": {
-        "php": ">=5.3",
-        "squizlabs/php_codesniffer": "~1.0"
+    "autoload": {
+       "psr-4": {
+           "Doctrine\\Sniffs\\": "lib/Doctrine/Sniffs"
+       }
     },
-    "require-dev": {
-        "phpunit/phpunit": "~4.7"
+    "require": {
+        "php": "^7.1",
+        "squizlabs/php_codesniffer": "~3.0",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.4"
```

Comment:
The other option is to manually have composer scripts to register the directory (on each repo).